prometheus/setup.cfg

```
@@ -0,0 +1,32 @@
+[metadata]
+name = indico-plugin-prometheus
+version = 3.2-dev
+description = Prometheus metrics in Indico servers
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8; variant=GFM
+url = https://github.com/indico/indico-plugins
+license = MIT
+author = Indico Team
+author_email = indico-team@cern.ch
+classifiers =
+    Environment :: Plugins
+    Environment :: Web Environment
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+
+[options]
+packages = find:
+zip_safe = false
+include_package_data = true
+python_requires = >=3.9.0, <3.11
+install_requires =
+    indico>=3.2
+    prometheus-client
```

```suggestion
    prometheus-client==0.16.0
```

they have a low release frequency and with 0.x versions i wouldn't be surprised of breaking changes between releases